Corona-Warn-App/src/main/res/navigation/nav_graph.xml

```diff
@@ -474,6 +478,12 @@
             app:argType="boolean" />
     </fragment>
 
+    <action
+        android:id="@+id/action_submissionConsentFragment"
+        app:destination="@id/submissionConsentFragment"
+        app:popUpTo="@id/universalScanner"
+        app:popUpToInclusive="true" />
```

Comment:
Universal scanner is not opened  in this case , does it ? should not that be  the `mainFragment`?

Reply:
Yes, you're right. Didn't catch that.